demo/core_40_world.js

```diff
@@ -74,18 +74,26 @@
   };
   clock.moveTo(hangout);
   clock.chime = function(silent) {
+    var minuteMs = 60 * 1000;
+    var hourMs = 60 * minuteMs;
```

Comment:
I'd have written msPerMinute and msPerHour, just to avoid having Megaseconds in my var name...